src/main/java/Papagu/Ui/Parser.java

```diff
@@ -0,0 +1,233 @@
+package Papagu.Ui;
+
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.time.LocalTime;
+
+/**
+ * Deals with making sense of user command
+ */
+
+public class Parser {
+    /**
+     * Function to help convert month to int format
+     * @param month
+     * @return
+     */
+    private static int monthConverter(String month) {
+        switch (month) {
```

Comment:
I think this does not follow the coding standards, as switch and case should have the same level of indentation